lib/screens/explore_screens/explore/explore_screen.dart

```diff
@@ -60,7 +60,7 @@ class ExploreScreen extends StatelessWidget {
       ExploreItem(
         title: context.loc.explorerUnplantItemTitle,
         icon: const CustomPaint(size: Size(31, 41), painter: PlantSeeds()),
-        onTapEvent: const OnExploreCardTapped(Routes.unPlantSeeds),
+        onTapEvent: const OnExploreCardTapped(Routes.plantSeeds),
```

Comment:
is this change expected?

Reply:
nop, I use this to show my screens and added plant instead of unplant by mistake.

Reply:
fixed